test/shared/src/test/scala/monocle/TestInstances.scala

```diff
@@ -25,38 +23,25 @@ trait TestInstances extends PlatformSpecificTestInstances with cats.instances.Al
   }
 
   // Equal instances
-  implicit val uriEqual        = Equal.fromUniversalEquals[URI]
+  implicit val uriEqual        = Eq.fromUniversalEquals[URI]
 
-  implicit val aritiesEq       = Equal.fromUniversalEquals[Arities]
-  implicit val nullaryEq       = Equal.fromUniversalEquals[Nullary]
-  implicit val unaryEq         = Equal.fromUniversalEquals[Unary]
-  implicit val binaryEq        = Equal.fromUniversalEquals[Binary]
-  implicit val quintaryEq      = Equal.fromUniversalEquals[Quintary]
+  implicit val aritiesEq       = Eq.fromUniversalEquals[Arities]
+  implicit val nullaryEq       = Eq.fromUniversalEquals[Nullary]
+  implicit val unaryEq         = Eq.fromUniversalEquals[Unary]
+  implicit val binaryEq        = Eq.fromUniversalEquals[Binary]
+  implicit val quintaryEq      = Eq.fromUniversalEquals[Quintary]
 
-  implicit def optionCofreeEq[A](implicit A: Equal[A]): Equal[Cofree[Option, A]] =
-    Equal.instance { (a, b) =>  A.eqv(a.head, b.head) && a.tail === b.tail }
+  implicit def optionCofreeEq[A](implicit A: Eq[A]): Eq[Cofree[Option, A]] =
+    Eq.instance { (a, b) =>  A.eqv(a.head, b.head) && a.tail === b.tail }
 
-  implicit def streamCofreeEq[A](implicit A: Equal[A]): Equal[Cofree[Stream, A]] =
-    Equal.instance { (a, b) =>  A.eqv(a.head, b.head) && a.tail === b.tail }
+  implicit def pisoEq[S, T, A, B](implicit StoA: Eq[S => A], BtoT: Eq[B => T]): Eq[PIso[S, T, A, B]] =
```

Comment:
I am not sure why the test with Iso/Prism equality do not compile, you seem to have kept these test instances